### PR TITLE
Add nd2reader recipe

### PR DESCRIPTION
**Pull request by caspervdw**
See https://github.com/rbnvrw/nd2reader  @rbnvrw @jimrybarski

### CODE REVIEW

File: recipes/nd2reader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  description: |
+    nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+. It has 
+    only been definitively tested on NIS Elements 4.30.02 Build 1053. Support for older versions 
+    is being actively worked on. The reader is written in the pims framework, enabling easy 
+    access to multidimensional files, lazy slicing, and nice display in IPython.
+
+extra:
+  recipe-maintainers:
+    - rbnvrw
```
Comment (synapticarbors) [inline recipes/nd2reader/meta.yaml]: @rbnvrw - are you ok with being added as a co-maintainer of the nd2reader package on conda-forge?

File: recipes/nd2reader/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+about:
+  home: https://github.com/rbnvrw/nd2reader
+  license: GPL-3.0
+  summary: 'Pure Python library for reading NIS Elements ND2 images and metadata'
+
+  description: |
+    nd2reader is a pure-Python package that reads images produced by NIS Elements 4.0+. It has 
+    only been definitively tested on NIS Elements 4.30.02 Build 1053. Support for older versions 
+    is being actively worked on. The reader is written in the pims framework, enabling easy 
+    access to multidimensional files, lazy slicing, and nice display in IPython.
```
Comment (synapticarbors) [inline recipes/nd2reader/meta.yaml]: If this page is still up-to-date, please add:

```
doc_url: http://www.lighthacking.nl/nd2reader/
```

File: recipes/nd2reader/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - xmltodict >=0.9.2
+    - pims >=0.3.0
+
+test:
+  imports:
+    - nd2reader
+
+about:
+  home: https://github.com/rbnvrw/nd2reader
+  license: GPL-3.0
```
Comment (synapticarbors) [inline recipes/nd2reader/meta.yaml]: Please submit an issue or PR upstream to add a `MANIFEST.in` to the source repo that will force the `LICENSE.txt` file distributed with the package. 

For GPL, we must distribute the license, so until the above happens in a future release, please copy the `LICENSE.txt` file into the recipe directory and add the following:

```
license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

See:
http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

and an example:

https://github.com/conda-forge/wsaccel-feedstock/tree/master/recipe

File: recipes/nd2reader/meta.yaml (hunk 2 above)
Comment (rbnvrw) [inline recipes/nd2reader/meta.yaml]: Yes, this page is up to date. Can you add it @caspervdw ?

File: recipes/nd2reader/meta.yaml (hunk 1 above)
Comment (rbnvrw) [inline recipes/nd2reader/meta.yaml]: Yes, thank you!

File: recipes/nd2reader/meta.yaml (hunk 2 above)
Comment (caspervdw) [inline recipes/nd2reader/meta.yaml]: Done

File: recipes/nd2reader/meta.yaml (hunk 3 above)
Comment (caspervdw) [inline recipes/nd2reader/meta.yaml]: Done, thanks for the help @synapticarbors

File: recipes/nd2reader/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.6.2
```
Comment (synapticarbors) [inline recipes/nd2reader/meta.yaml]: Sorry for adding another request, but I unfortunately forgot it in my initial review. When you use `--single-version-externally-managed` you don't need to specify what is included in `install_requires` in the `setup.py`, unless `setup.py` actually attempts to import them or something inside the package that does. So I think you only need `python` and `setuptools` in the build section.

Comment (caspervdw) [inline recipes/nd2reader/meta.yaml]: Sure, thanks for the feedback. I didn't know this, I'll push the change soon.

**Pull request merged**